CRM/Utils/System/Base.php

```diff
@@ -718,7 +718,8 @@ public function getCiviSourceStorage() {
       // the system for a loop on lobo's macosx box
       // or in modules
       $cmsPath = $config->userSystem->cmsRootPath();
-      $userFrameworkResourceURL = $baseURL . str_replace("$cmsPath/", '',
+      $realCmsPath = realpath($cmsPath);
+      $userFrameworkResourceURL = $baseURL . str_replace(["$cmsPath/", "$realCmsPath/"], '',
```

Comment:
```suggestion
      $realCmsPath = realpath($cmsPath);

      if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
        $cmsPath = strtolower(str_replace('\\', '/', $cmsPath));
        $realCmsPath = strtolower(str_replace('\\', '/', $realCmsPath));
      }

      $userFrameworkResourceURL = $baseURL . str_replace(["$cmsPath/", "$realCmsPath/"], ['', ''],
```

Reply:
The fix has a significant impact under Windows. For example, without it, URLs to JS resources have the format `http: //myproject.local/c:/projects/myproject/sites/all/modules/civicrm/packages/jquery/plugins/jquery.timeentry.min.js`.
Below debug of key variables, please pay attention to slash, backslahe, upper and lower case (hard drive letter):
```
$civicrm_root => c:/projects/myproject/sites/all/modules/civicrm/
$cmsPath => C:/xampp-702/htdocs/myproject
$realCmsPath => C:\projects\myproject
```